### PR TITLE
Release v1.33.0

### PR DESCRIPTION
**Pull request by Kalaiselvi84**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind release

**What this PR does / Why we need it**:

**Which issue(s) this PR fixes**:

Work on #3246

**Special notes for your reviewer**:

### CODE REVIEW

**Review by gongmax — APPROVED** [review]
LGTM

File: site/content/en/blog/releases/1.33.0.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+This is the 1.33.0 release of Agones.
+
+In this release, we have focused on improving the documentation, stabilized the SDKGracefulTermination feature and also are now returning GameServer annotations and labels on allocation response.
+
+- Stable SDKGracefulTermination: 
+    SDKGracefulTermination feature is promoted to stable. With this advancement, the SDK server will now remain active for the specified `terminationGracePeriodSeconds` until `SDK.Shutdown()` has been called. This ensures a seamless transition during shutdown scenarios and gives you more control over the termination process.
+
+- Enhanced Documentation: 
+	We have taken steps to improve the usability of our [helm documentation](https://github.com/googleforgames/agones/blob/05ab9ca00ece135abae21611a67344777b30dc36/site/content/en/docs/Installation/Install%20Agones/helm.md). The large configuration table has been restructured into multiple sections making it much easier to navigate the Agones services.
```
Comment (roberthbailey) [inline site/content/en/blog/releases/1.33.0.md:15]: Can you change the link to https://agones.dev/site/docs/installation/install-agones/helm/#configuration?

**Review by roberthbailey — APPROVED** [review]
Generally looks good, but please fix the hyperlink and wait for Mark to review the changes to helm.md.

File: site/content/en/blog/releases/1.33.0.md (hunk 1 above)
Comment (Kalaiselvi84) [reply site/content/en/blog/releases/1.33.0.md]: I have modified the helm-configuration link as per your suggestion.

**Review by markmandel — APPROVED** [review]
LGTM! Had a very small nit below, but marking as approved so you can run through without needing further input.

File: site/content/en/blog/releases/1.33.0.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+- Enhanced Documentation: 
+	We have taken steps to improve the usability of our [helm documentation](https://agones.dev/site/docs/installation/install-agones/helm/#configuration). The large configuration table has been restructured into multiple sections making it much easier to navigate the Agones services.
+
+- Returning GameServer labels and annotations to allocation response:
+	The GameServerAllocation now has a metadata field returned with the metadata of the allocated GameServer. This can be useful for returning information about the GameServer, Fleet or even the cluster to the allocating process.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.33.0" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Implemented enhancements:**
```
[the remark this answers is not in the document]
Comment (Kalaiselvi84) [reply site/content/en/blog/releases/1.33.0.md:22]: added only enhancements👍🏻

File: site/content/en/docs/Installation/_index.md [hunk 3]
```diff
@@ -47,7 +47,8 @@ The following table lists recent Agones versions and their corresponding require
 
 | Agones version | Kubernetes version(s) |
 | -------------- | ------------------    |
-| 1.32           | {{% k8s-version %}}   |
+| 1.33           | {{% k8s-version %}}   |
+| 1.32           | 1.24, 1.25, 1.26   |
```
[the remark this answers is not in the document]
Comment (Kalaiselvi84) [reply site/content/en/docs/Installation/_index.md]: Good eye, missed the alignment, sorry..

**Pull request merged**